cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2325,7 +2332,15 @@ function start-kube-controller-manager {
     params+=" --use-service-account-credentials"
   fi
   params+=" --cloud-provider=gce"
-  params+=" --kubeconfig=/etc/srv/kubernetes/kube-controller-manager/kubeconfig"
+  ## hack, to workaround a RBAC issue with the controller token, it failed syncing replicasets so pods cannot be created from the deployments
+  ## TODO: investigate and fix it later
+  #
+  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "false" ]]; then
```

Comment:
here as well

Reply:
will address along with the decision 1) name of the variable to controller the insecure/secure port  2) two variables: one for enable insecure port, one for use insecure ports in kubemarktest